### ISSUE TITLE
Installing SAP HANA 2.0, express edition (Binary Installer Method) - installation of cockpit-collection-svc fail

### ISSUE THREAD

**Issue by timemaster67** [SAPDocuments/Tutorials#1966]
Hello, 
I have followed the tutorial and downloaded the server and applications packages. The installation of the cockpit always fail. 
I have been trying this installation 3 times and it always fail at the same place. 
step to reproduce
1. extract the server only tgz, and then extract the application tgz
2. Edit configuration files
2.1 navigate to HANA_EXPRESS_20/DATA_UNITS/HDB_LCM_LINUX_X86_64/configurations
2.2 edit auto_install.cfg, change property  to
datapath=/hana/data/${sid}
logpath=/hana/log/${sid}
3. start installation by executing setup_hxe.sh
3.1 specify instance number 00 instead of default 90. 

installation of sap cockpit fail. 
Final error on the console and setup_hxe is 

Uploading application "cockpit-collection-svc"...
2017-10-13 14:59:15 :     Installation of archive file(s) '/hana/REDACTED1/SAPHANA2SP02/Binary_Installer/HANA_EXPRESS_20/DATA_UNITS/HANA_COCKPIT_20/XSACCOCKPIT03_7.zip' failed.
2017-10-13 14:59:20 :   Installation of SAP HANA Cockpit failed 

final error in hdblcm is
Output line 150: Downloaded 'SAP JVM Memory Calculator', version '1.6.1' in 0.1 s.
14:52:57.417 - INFO:       Output line 151: Application "cockpit-collection-svc" staged
14:59:08.459 - INFO:       Output line 152: ERR    Error occured during communication with HTTP server
14:59:12.471 - INFO:       Output line 153: 
14:59:15.015 - INFO:       Output line 154: Installation of archive file(s) '/hana/REDACTED1/SAPHANA2SP02/Binary_Installer/HANA_EXPRESS_20/DATA_UNITS/HANA_COCKPIT_20/XSACCOCKPIT03_7.zip' failed.

Installation was done on a suse linux enterprise server 12 SP2. 
[hdblcm.log](https://github.com/SAPDocuments/Tutorials/files/1383844/hdblcm.log)
[setup_hxe_2017-10-13_13.39.56.log](https://github.com/SAPDocuments/Tutorials/files/1383845/setup_hxe_2017-10-13_13.39.56.log)

**Comment by Lsubatin** [comment]
Hi, 

Some questions for you:

1- Where are steps 2 and 3 coming from? 
2- Also, how much RAM does your machine have? 3
3- Can you try again and execute /usr/sap/HXE/home/bin/hxe_gc,sh in a separate terminal when it's starting the cockpit installation?

**Comment by timemaster67** [comment]
Hello @Lsubatin, 
while you did not clearly gave the answer needed, your poste was useful because you seemed to imply that the machine running the installation did not have enough ram. 

It was the case, the machine did not have enough ram. By using the hxe_gc.sh - which basically try to scan the memory of hana and try to free reserved but unused memory, I was able to install the xsa packages. 

There was not was not enough ram assigned to be virtual machine because I did not see that the binary installer and the virtual machine requirements were different. For other people that might find this bug : 

requirement for virtual machine template to restore : 
Server plus applications virtual machine:●
16 GB RAM minimum. 24 GB RAM recommended.
Server-only virtual machine:●
8 GB RAM minimum. (If you add additional components, or run heavy processing loads, you will need to increase your RAM.)

requirement for binary installer : 
Server: 16 GB minimum (24 GB recommended)
Server + Applications: 24 GB minimum (32 GB recommended)Note
If you are installing on a system with 16 GB of RAM, increase the amount of swap space to at least 32 GB.
